addons/phone_validation/lib/phonenumbers_patch/region_FR.py

```diff
@@ -0,0 +1,21 @@
+"""Auto-generated file, do not edit by hand. FR metadata"""
+from ..phonemetadata import NumberFormat, PhoneNumberDesc, PhoneMetadata
+
+PHONE_METADATA_FR = PhoneMetadata(id='FR', country_code=33, international_prefix='00',
+    general_desc=PhoneNumberDesc(national_number_pattern='[1-9]\\d{8}', possible_length=(9,)),
+    fixed_line=PhoneNumberDesc(national_number_pattern='(?:26[013-9]|59[1-35-9])\\d{6}|(?:[13]\\d|2[0-57-9]|4[1-9]|5[0-8])\\d{7}', example_number='123456789', possible_length=(9,)),
+    mobile=PhoneNumberDesc(national_number_pattern='(?:[67](?:[0-24-8]\\d|3[0-8]|9[589])|7[3-9]\\d)\\d{6}', example_number='612345678', possible_length=(9,)),
```

Comment:
if the intent of the change of the original file:

```diff
-    mobile=PhoneNumberDesc(national_number_pattern='(?:6(?:[0-24-8]\\d|3[0-8]|9[589])|7[3-9]\\d)\\d{6}', example_number='612345678', possible_length=(9,)),
+    mobile=PhoneNumberDesc(national_number_pattern='(?:[67](?:[0-24-8]\\d|3[0-8]|9[589])|7[3-9]\\d)\\d{6}', example_number='612345678', possible_length=(9,)),
```

 is to add numbers beginning by 70 to 72, it would be cleaner to do it like this:

```suggestion
    mobile=PhoneNumberDesc(national_number_pattern='(?:6(?:[0-24-8]\\d|3[0-8]|9[589])|7[0-9]\\d)\\d{6}', example_number='612345678', possible_length=(9,)),
```